test/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -150,11 +191,11 @@ class RedirectVerifier(val response: IdpRedirectResponse) : BindingVerifier() {
     }
 
     /**
-     * Verifies the destination is correct according to the redirect binding rules in the bindinc spec
+     * Verifies the destination is correct according to the redirect binding rules in the binding spec
      * 3.4.5.2 Security Considerations
      */
     private fun verifyRedirectDestination() {
-        val destination = response.responseDom.attributes.getNamedItem("Destination")?.nodeValue
+        val destination = response.responseDom!!.attributes.getNamedItem("Destination")?.nodeValue
```

Comment:
❗️ This can potentially through an NPE (protection against which is, please recall, one of the primary advantages that Kotlin provides over Java).

See note on `IdpResponseDecorators.kt`.